crates/consensus/src/transaction/eip2930.rs

```diff
@@ -29,8 +29,8 @@ pub struct TxEip2930 {
     /// this transaction. This is paid up-front, before any
     /// computation is done and may not be increased
     /// later; formally Tg.
-    #[cfg_attr(feature = "serde", serde(with = "alloy_serde::u64_hex_or_decimal"))]
-    pub gas_limit: u64,
+    #[cfg_attr(feature = "serde", serde(with = "alloy_serde::u128_hex_or_decimal"))]
```

Comment:
do any endpoints use decimal here? in other PRs we've avoided allowing decimal when nobody actually uses it